tests/features/support/steps.js

```diff
@@ -207,6 +215,59 @@ Then(/the link "([^"]+)" should open in (a new window|the same window)/, async (
   await scope.afterStep(scope);
 });
 
+// FEATURE IN DEVELOPMENT. A FOOL'S ERRAND
+let number_map = {first: 0, second: 1, third: 2, fourth: 3, fifth: 4, sixth: 5, seventh: 6, eighth: 7, ninth: 8, tenth: 9, };
+let specified = '?"?(first|second|third|fourth|fifth|sixth|seventh|eighth|ninth|tenth|"[^"]+)?"?';
+let prohibited_choice_words = `I should not see the options? "([^"]+)" ?(?:in the ${specified} choice list)?`;
+let prohibited_choice_words_regex = new RegExp(prohibited_choice_words);
+Then(prohibited_choice_words_regex, async (prohibited, specifier) => {
+  /* In development.
+  *
+  * Checks that `prohibited` does not appear in the text of the
+  *    the list of options in the <select> with the label "containing" the `label text`.
+  *    Only checks the first <select>.
+  *
+  * @param prohibited {string} String, possibly containing commas and ', or' to
+  *    denote multiple phrases, that should not appear in the options.
+  * @param specifier {string} Text that is in the label of the <select>
```

Comment:
Mention that this is an optional param.

Reply:
Also, maybe mention that the specifier will be some sort of ordinal number (e.g., first, third, fifth). That's not immediately obvious to me.

Reply:
It can also be a string, but you're right that I failed to update that spec. Will do 👍